api/tests/integration/AuthTest.php

```diff
@@ -106,6 +91,7 @@ public function testLoginUserMissCredentials()
 
     public function testFailedTokenEncoding()
     {
+        $this->markTestSkipped('why check that?');
```

Comment:
Well if we don't have a specific section of code that is catching the namshi exception and formatting then sure, we don't need it. Remove the test if that's the case.